man/systemd-analyze.xml

```diff
@@ -284,6 +284,17 @@ Timestamp units-load-finish: Thu 2019-03-14 23:28:07 CET
 $ eog bootup.svg&amp;
 </programlisting>
       </example>
+
+      <para>Note that this plot is based on the most recent per-unit timing data of loaded units. This means
+      if – after boot – a unit gets started, then stopped and then started again the information shown will
```

Comment:
"This means *that* if"

Hmm, why "after boot"? If it gets stopped and started again during boot, we would still show only info about the latest invocation, iiuc.